File: Most-Recursion/factorial.py
def fact(n): # recursion function
    if(n == 1): # base case
        return 1
    return n*fact(n-1) # call it's self
print(fact(5),'python')


# Itrative working
fac = 1
n = 5
for i in range(1,n+1):
    fac = fac*i
print(fac,'itrative')

File: Most-Recursion/factorial.js
function fact(n){ // recursion function
    if(n == 1) // base case 
        return 1;
    return n*fact(n-1); // call it's self
}
console.log(fact(5),'javascript');


// Iterative working
let n = 5;
let fac = 1;
for(i=5; i>=1; i--){
    fac = fac*i;
}
console.log(fac);